homeassistant/components/isy994/cover.py

```diff
@@ -49,16 +64,28 @@ def is_closed(self) -> bool:
             return None
         return self._node.status == 0
 
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
+
     def open_cover(self, **kwargs) -> None:
         """Send the open cover command to the ISY994 cover device."""
-        if not self._node.turn_on(val=100):
+        position = kwargs.get(ATTR_POSITION)
```

Comment:
It doesn't look like ATTR_POSITION is ever passed to `open_cover`.   This should probably be implemented in `set_cover_position` instead.

https://www.home-assistant.io/integrations/cover/

Reply:
I was just relying on a cascade of `kwargs` from `set_cover_position` to `open_cover`.  Normal turn_on commands would just result in `None` and pass that as `val=` which is fine from the PyISY side. I can just duplicate the call in `set_cover_position`.

Reply:
That's better as we don't have open accepting `ATTR_POSITION` which could lead other people copying code down the wrong path.